woocommerce/woocommerce
# Update turborepo to latest version 1.8.3

### Changes proposed in this Pull Request:

This updates TurboRepo to 1.8.3 which is the [latest version](https://github.com/vercel/turbo/releases/tag/v1.8.3).

I can't quantify this with much data, but when running into issues with build order of TS modules recently, I noticed that updating to TurboRepo 1.8.3 seemed to fix these problems immediately.


### How to test the changes in this Pull Request:

Do a smoke test of build locally:

1.  Pull down this branch
2. Run pnpm i (this should run anyway when you pull down the branch)
3. Run pnpm build
4. Build should pass

This and CI passing should be enough confidence that its working.






### FOR PR REVIEWER ONLY:
